var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -110,8 +110,27 @@ def install(self, spec, prefix):
                         '-rhsone')
             make("install")
 
+    @property
+    def headers(self):
+        """Export the main hypre header, HYPRE.h; all other headers can be found
+        in the same directory.
+        Sample usage: spec['hypre'].headers.cpp_flags
+        """
+        hdrs = HeaderList(find(self.prefix.include, 'HYPRE.h',
```

Comment:
I would just use `find_headers` instead.

Reply:
OK, I'll switch to `find_headers`.